STANDARD.md

```diff
@@ -48,12 +48,14 @@ Use these standard notations for writing pseudocodes.
   ```plaintext
   while condition:
       # code block
-
-  for i = startValue to endValue:
+  
+  for i = startValue to endValue step stepValue:
       # code block
   
-  for i = startValue to endValue step 2:
+  # For stepValue = 1
+  for i = startValue to endValue:
       # code block
```

Comment:
If we use `for` loop, no matter what! we should add `stepValue`.
So, we don't need second `for` loop.

Reply:
That will make the loop lengthy without any reason. We will not use step value apart from some special cases. So it is better to have a plain version, that uses single steps.

Reply:
Nope. We will have to make one standard and no matter what we should have to stick with that standard.